bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/devices/bridge/dto/PublicInformation.java

```diff
@@ -42,4 +42,9 @@ public PublicInformation() {
     public List<String> apiVersions;
     public String shcIpAddress;
     public String shcGeneration;
+    public SoftwareUpdateState softwareUpdateState;
+
+    public static Boolean isValid(PublicInformation obj) {
```

Comment:
```suggestion
    public static boolean isValid(PublicInformation obj) {
```

Also it feels a bit strange to have a `isValid` static method that only checks for nullness. Is this method for future expansion?

Reply:
If I see correctly, the return type is still `Boolean` instead of `boolean`. I don't see why we would need a `Boolean` object instead of a primitive `boolean`.

Reply:
Yes, @david-pace up to now I only change the implementation.
I'm sorry to say that I'm not able to provide a normal, expected development speed and flow. I need weeks and not days.
Because I saw that other dto classes use also ``Boolean`` types I want to update all places which takes more time than just to change the single word. Please give me still some days more, even the new update is not a really big one.

Reply:
changed to boolean